daemon/daemon_main.go

```diff
@@ -770,6 +773,10 @@ func init() {
 	flags.Bool(option.EnableAPIRateLimit, false, "Enables the use of the API rate limiting configuration")
 	option.BindEnv(option.EnableAPIRateLimit)
 
+	flags.Bool(option.EgressMultiHomeIPRuleCompat, false,
+		"Use a new scheme to store rules and routes under ENI and Azure IPAM modes, if false. Otherwise, it will use the old scheme.")
```

Comment:
nit: see nit about flag description in generated `Documentation/cmdref/cilium-agent.md` above.